scripts/lighthouse/runLighthouse.js

```diff
@@ -0,0 +1,58 @@
+const lighthouse = require('lighthouse'); // eslint-disable-line import/no-extraneous-dependencies
+const chromeLauncher = require('chrome-launcher'); // eslint-disable-line import/no-extraneous-dependencies
+
+let config = {};
+
+function launchChromeAndRunLighthouse(url, opts, lConfig = null) {
+  const options = opts;
+  return chromeLauncher
+    .launch({ chromeFlags: options.chromeFlags })
+    .then(chrome => {
+      options.port = chrome.port;
+      return lighthouse(url, options, lConfig).then(results =>
+        chrome.kill().then(() => results.lhr),
+      );
+    });
+}
+
+function getScoresPerCategory(reportCategories, url) {
+  const scores = Object.keys(reportCategories).map(category => ({
+    id: reportCategories[category].id,
+    score: reportCategories[category].score,
+  }));
+
+  return { scores, url };
```

Comment:
Generally I'd try to just `return scores` - returning an object suggests a refactor would be good somewhere.

Reply:
The reasoning behind how it is is that Lighthouse results don't give a tested URL with the results object; the lighthouse scripts are breaking down score per URL because they could vary as scenarios vary (especially when we test more than two scenarios, with different combinations of components).